src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/transformation/model/request/ExceptionRecord.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.client.transformation.model.request;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Classification;
```

Comment:
Single import used outside of `client.transformation.model.request` compared to other copies